machineProblem.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+import sys
+import tkinter
+from tkinter import messagebox
+from tkinter import font
+
+
+root = tkinter.Tk()
+root.title(u"Who Wants to Be A Millionaire.")
+root.geometry("1280x720")
+
+#ラベル
+Static1 = tkinter.Label(text=u'Who Wants to Be A Millionaire.', font=font.Font(size=37))
+Static1.pack()
+
+
+win = tkinter.Toplevel()
+frame = tkinter.Frame(master=win).grid(row=1, column=1)
+
+button1 = tkinter.Button(master=frame, text=u'PLAY', width=30,                      command=lambda: controller.show_frame(yourName))
```

Comment:
controller is not defined here. Instead, forget defining controller, and put it as such:
command=lambda: show_frame(yourName)

also try minding the space too :)